virtio-devices/src/vhost_user/fs.rs

```diff
@@ -78,11 +78,12 @@ impl VhostUserMasterReqHandler for SlaveReqHandler {
             }
 
             let addr = self.mmap_cache_addr + offset;
+            let flags = fs.flags[i];
```

Comment:
Why the "unaligned" problem can be resolved by simply splitting the single expression into two parts? In my opinion, nothing was changed this way.

This is not a change request. I just wonder because I also looked into this.

Reply:
You're now taking the reference (via `&self` inside the `.bits()`) on a copy of the value that was copied from the packed struct. Therefore you are no longer taking a reference to something that is ultimately inside the packed struct.

Reply:
I see. Thank you very much.